### ISSUE TITLE
Polylang does not fully follow the setting "no trailing slash" in URLs

### ISSUE THREAD

**Issue by jasomdotnet** [polylang/polylang#1478]
I work on website migration where in original URLs is no ending backslash `/`. To keep the links as intact as possible I set no trailing slash at the end in new WP installation. You can do it easily like [here(https://stackoverflow.com/questions/8558234/remove-trailing-slash-using-htaccess-in-wordpress). 

Polylang does not fully follow the setting. When I open the home page of a locale like `https://www.example.com/en/` the plugin appends `/` to the end of the URL even though no slash is set. Expected URL is `https://www.example.com/en`.

Can you synchronize Polylang's current behavior with this Wordpress "no ending backslash" setting?

**Comment by Chouby** [comment]
This is by design. Polylang follows the same rule as WordPress. When the home url includes a directory, then WordPress always adds a trailing slash, even when permalinks options don't include one. This is not the case if the home url includes only the domain name, but it's visible when WordPress is installed in a subdirectory or on mutisites.

**Comment by jasomdotnet** [comment]
I did not get from your what is "by design". What the word "this" if first sentence stands for? Any way, ones again.

In WP installation I set URL not to use `/` at the end if permaling. Despite this setting, Polylang plugin add `/` at the end of localized homepage like `https://www.example.com/en/`.

This is the problem:
> When the home url includes a directory, then WordPress always adds a trailing slash, even when permalinks options don't include one.

It did not get the last sentence 
> This is not the case if the home url includes only the domain name, but it's visible when WordPress is installed in a subdirectory or on mutisites.

but in my opinion it doesn't matter within this issue, because installation in question is neither installed in a subdirectory not is it a multisite.

Localized homepage URL should be `https://www.example.com/en` when set so in permaling options.

Polylang is great plugin. Good job, anyway.
